receiver/kafkametricsreceiver/README.md

```diff
@@ -15,6 +15,13 @@
 Kafka metrics receiver collects kafka metrics (brokers, topics, partitions, consumer groups) from kafka server,
 converting into otlp.
 
+## Prerequisites
+
+This receiver supports Kafka versions:
+  -  2.0+
+  -  3.0+
```

Comment:
Would it be more accurate to say `2.X` and `3.X`? Once `4.0` is released it may not be 100% clear if it's supported or not, so we may want to be a bit more clear.

Reply:
I have not tested 4.0 or above to make that statement.  By looking at their other receivers ( for e.g.) I assume 3.0+ means 3.x ( for e.g. 3.0, 3.1, 3. 5 so on).